strings.go

```diff
@@ -31,6 +31,9 @@ const CHANNEL_DOES_NOT_EXIST_MSG = "The channel you specified does not exist."
 // Message shown to users when they attempt to add an invalid URL to the queue.
 const INVALID_URL_MSG = "The URL you submitted does not match the required format."
 
+// Message shown to users when they attempt to search on an invalid platform.
+const INVALID_SEARCH_PROVIDER  = "The Search provider you submitted does not match the required format."
```

Comment:
This might be a better way to state this message: 

"The service you tried to search for an item on is not supported. Supported services are youtube and soundcloud."